src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
According to the checklist, gaps should be defined by multiples of 4px. The current gap of 1px does not follow this rule.